Phase-5/SearchWebAPI/Controllers/SearchController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using SearchEngine.Classes.IO.Database.Models;
+using SearchEngine.Interfaces.Core;
```

Comment:
I Actually really love this comment:
useless imports :)))))) (just kidding)
it is not really important but it is good to remove useless imports